css/base.styl

```diff
@@ -154,6 +154,11 @@ html.has-mozilla .dgrid-focus {
 	top: -9999px;
 }
 
+.dgrid-scrollbar-measure-child {
+  width: 10px;
+  height: 10px;
+}
+
```

Comment:
Spaces, not tabs! (Stylus generates spaces when it outputs non-compressed CSS, which is likely where you copied this from since you had edited the CSS first.  I'll fix this when merging.)